lib/remap-functions/src/log.rs

```diff
@@ -23,6 +23,11 @@ impl Function for Log {
                 accepts: |v| matches!(v, Value::Bytes(_)),
                 required: false,
             },
+            Parameter {
+                keyword: "rate_limit_secs",
+                accepts: |v| matches!(v, Value::Integer(_)),
```

Comment:
I think the unit `seconds` is fine, but I feel like we should also be able to set the rate limiting in sub-seconds `0.1` or other fractions `1.5`.

```suggestion
                accepts: |v| matches!(v, Value::Float(_)),
```